rust/lon/src/lock/v1.rs

```diff
@@ -33,6 +33,7 @@ pub struct GitSource {
     pub revision: String,
     pub url: String,
     pub hash: SriHash,
+    pub last_modified: u64,
```

Comment:
Let's make this optional then we don't have to increment the version of the lockfile.